Documentation/ApiOverview/Backend/ContextualMenu.rst

```diff
@@ -31,15 +22,18 @@ Context menu rendering flow
 Markup
 ------
 
-The context menu is shown after click on the HTML element which has
+The context menu is shown after clicking on the HTML element which has
 :html:`class="t3js-contextmenutrigger"` together with :html:`data-table`,
 :html:`data-uid` and optional :html:`data-context` attributes.
 
 The JavaScript click event handler is implemented in the
-:js:`TYPO3/CMS/Backend/ContextMenu` requireJS module. It takes the
-data attributes mentioned above and executes an ajax call to the
+ES6 module :js:`@typo3/backend/context-menu.js`. It takes the
+data attributes mentioned above and executes an AJAX call to the
```

Comment:
Wikipedia always writes it "Ajax" also mentioning it my be written "AJAX". It think it is getting just like a word, as the X does not stand anymore for XML but can also by JSON or whatever:
https://en.wikipedia.org/wiki/Ajax_(programming)